web/sites/default/settings.lando.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+// @codingStandardsIgnoreFile
+
+/**
+ * @file
+ * Drupal site-specific configuration file.
+ */
+
+// Lando environment settings
+if (getenv('LANDO_INFO')) {
```

Comment:
This check is already done when seeing if the file should be included

Reply:
True. Out of habit, I always check variabless before referencing them -- but you're correct that this should _never_ be reached if the var is not defined. It's redundant.

Reply:
I do not think Lando has a standard. For consistency with DDEV, I will keep the check here and remove the one in `settings.php`.